src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -329,6 +329,8 @@ ClientRequest::process_op(
     }
   }
 
+  co_await ihref.enter_stage<interruptor>(
```

Comment:
```
/**
  * The previous stage of recover_missing is a concurrent phase.
  * Checking for already_complete requests must done exclusively.
  * Since get_obc is also an exclusive stage, we can merge both stages into
  * a single stage and avoid stage switching overhead.
  */
```